docs/manual/java/guide/broker/KafkaClient.md

```diff
@@ -48,3 +51,26 @@ akka.kafka.producer.kafka-clients {
 }
 ```
 
+## Consuming only Services
+
+When implementing a Lagom Service that will only consume from a Kafka Topic you can import the Lagom Kafka Client alone (instead of importing the Lagom Kafka Broker and a Lagom Persistence implementation).
```

Comment:
I don't like the title to this section.  Does it sound correct in english? I'm trying to describe a service which uses kafka as a consumer only.